core/src/main/java/org/jruby/util/JRubyClassLoader.java

```diff
@@ -160,6 +166,10 @@ private static String systemTmpDir() {
         return "";
     }
 
+    public List<URL> getTempUrls(){
```

Comment:
would be great to have the naming a bit consistent with existing API e.g. `addURL(URL)`
... `getCachedURLs()` or maybe even indicate that there's temp file caching (that those URLs are actually local files)
in which case we should rather return String paths than raw URL, makes sense (`getTempPaths` or `getCachedPaths`)?

Reply:
looks much better than the current naming ... go for whatever you feel is appropriate.
getter sounds fine to me esp. if its simply returning a field